website/blog/2022-10-06-Ingest-streaming-data-to-Apache-Hudi-using-AWS-Glue-and-DeltaStreamer.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: "Ingest streaming data to Apache Hudi tables using AWS Glue and Apache Hudi DeltaStreamer"
+authors:
+- name: Vishal Pathak
+- name: Anand Prakash
+- name: Noritaka Sekiyama
+category: blog
+image: /assets/images/blog/2022-10-06_Ingest_streaming_data_to_Apache_Hudi_tables_using_AWS_Glue_and_DeltaStreamer.png
```

Comment:
Refers to the image checked in from this pr - https://github.com/apache/hudi/pull/6972